include/vapor/SliceParams.h

```diff
@@ -40,18 +39,19 @@ class PARAMS_API SliceParams : public RenderParams {
             return GetVariableName();
     }
 
-    int GetSampleRate() const;
-
-    void SetSampleRate(int rate);
-
-    int GetDefaultSampleRate() const;
-
-    void SetCachedValues(std::vector<double> values);
-
+    int                 GetSampleRate() const;
+    void                SetSampleRate(int rate);
+    int                 GetDefaultSampleRate() const;
+    void                SetCachedValues(std::vector<double> values);
     std::vector<double> GetCachedValues() const;
 
     static const string _sampleRateTag;
-    static const string SampleLocationTag;
+    static const string XRotationTag;
```

Comment:
Also, for consistency, why three component flags and not simply a vector RotationTag and an OrientationTag? This would be consistent with other vector quantities (e.g. RGB, domain extents, etc.)

Reply:
Single values (vs vectors) allow use of the low level interface.  No validation needs to be done for rotation on the X, Y, or Z axis.  So all we need is a single parameter for each rotation, and an LLI widget like what we have at [line 29 of SliceEventRouter.cpp](https://github.com/NCAR/VAPOR/blob/9fd7e90a3794383d826107c9a4160c3b1ddea680/apps/vaporgui/SliceEventRouter.cpp#L30).

This follows [the policy we wrote on PWidgets](https://docs.google.com/document/d/1LlTr1rHSRcCucN6U0KzW_HZUq_Q2cyPsQLD3nCJyf_k/edit#) a while back.